trace-forward/src/Trace/Forward/Protocol/Forwarder.hs

```diff
@@ -0,0 +1,63 @@
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE GADTs #-}
+{-# LANGUAGE LambdaCase #-}
+{-# LANGUAGE RankNTypes #-}
+{-# LANGUAGE RecordWildCards #-}
```

Comment:
I'd rather avoid `RecordWildCards` extendion; `NamedFieldPuns` is a better choice.

Reply:
Done.